website/static/js/components/legend.js

```diff
@@ -1,17 +1,56 @@
 require('css/legend.css');
 var m = require('mithril');
 
+/** Finds the index of a string in an array
+ *
+ * @param {Array} array The array to search through
+ * @param {String} str The string to find
+ * @returns {*}
+ */
+var find_index = function (array, str) {
+    for (var i = 0; i < array.length; i++) {
+        if (array[i].label === str)
+            return i;
+    }
+    return -1;
+};
+
+/** Removes an index from an array immutably
+ *
+ * @param {Array} array the array to modify
+ * @param {Int} index the index to remove
+ * @returns {*} a new array missing the index value
+ */
+var remove_index = function (array, index) {
+    var new_array = [];
+    for (var i = 0; i < array.length; i++) {
+        if (i !== index)
+            new_array.push(array[i]);
+    }
+    return new_array;
+};
+
 module.exports = {
-    view: function(data, repr, opts) {
-        if(data[0].label){
-            data.sort(function(a, b) {
+    view: function (data, repr, opts) {
+        var other_index = find_index(data, 'Other');
```

Comment:
Oh, okay... They already come last, but `projectOrganizerCategories` extends the sorted `CATEGORY_MAP` in an unsorted way, necessitating a sort. This is the passed-in ordering:

<img width="718" alt="screen shot 2015-12-18 at 10 29 53" src="https://cloud.githubusercontent.com/assets/5659262/11900431/ddfe4f24-a573-11e5-8e38-257e44fe7eb7.png">

`Collections`, `Smart Collections`, and `Link` aren't in `CATEGORY_MAP`. `Project` is but is also included in the `.extend` call, bumping it up.